### PR TITLE
Scala pride blog post!

### PR DESCRIPTION
**Pull request by heathermiller**

### CODE REVIEW

**Comment by heathermiller** [comment]
Fast-tracking this, happy to make changes in a subsequent PR if requested! 😊

File: blog/_posts/2018-06-20-scala-pride.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: blog-detail
+post-type: blog
+by: Heather Miller
+title: "Scala ❤️ Pride 2018"
+---
+
+It's that time of year! Pride 2018. In honor of all of us Scalamaris that are advocates of or identify with the LGTB community–we have a little bit of Scala love for you. I've created special Pride Scala swirls for just about every pride flag I could think of. And perhaps the best part! They're all available as t-shirts, available on [Teespring](http://teespring.com). 100% of proceeds for t-shirt sales are donated to [the Trevor Project](https://www.thetrevorproject.org/). (Go check it out!)
```
Comment (julienrf) [inline blog/_posts/2018-06-20-scala-pride.md]: LGBT (instead of LGTB)?

Comment (julienrf) [reply blog/_posts/2018-06-20-scala-pride.md]: Also “community-we have” sounds weird. Maybe you mean “community. We have”?

File: blog/_posts/2018-06-20-scala-pride.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+**But please move quick! The shirts are only available for around two weeks. So grab one or a few and help us raise funds for the [the Trevor Project](https://www.thetrevorproject.org/)!** Shirts are available in many colors and styles, and come as t-shirts or tank tops.
```
Comment (julienrf) [inline blog/_posts/2018-06-20-scala-pride.md]: “the [the Trevor Project]” (repetition of “the”)

File: blog/_posts/2018-06-20-scala-pride.md (hunk 1 above)
Comment (heathermiller) [reply blog/_posts/2018-06-20-scala-pride.md]: Goodness. SO MANY TYPOS. Thank you @julienrf!

Comment (heathermiller) [reply blog/_posts/2018-06-20-scala-pride.md]: In “community-we have”, the dash is actually an em dash. http://www.thepunctuationguide.com/em-dash.html

**Comment by SethTisue** [comment]
scalacenter.shop is 404 now; I removed the link in e1608e5551d983c0b56060eaf003e12b3f435009

**Pull request merged**